docs/src/components/home-preview.js

```diff
@@ -92,87 +88,6 @@ export const StyledPreview = styled(({ splitVertical, ...rest }) => (
   min-height: 25rem;
   overflow: hidden;
   flex-grow: ${({ splitVertical }) => (splitVertical ? 1 : 0)};
-
-  .mover {
```

Comment:
All of this was replaced by our root BEM-style selectors (all prefixed with `live-preview`). I figured keeping the CSS for all live previews co-located helped keep it more reusable and organized.